app/assets/javascripts/shared/SharedService.js.coffee

```diff
@@ -27,7 +27,7 @@ SharedService = ($http, $state, $window, $document) ->
   Service.getLanguageCode = (langName) ->
     # will take "English" and return "en", for example
     if langName
-      _.invert(Service.languageMap)[_.startCase(_.lowerCase(langName))]
+      _.invert(Service.languageMap)[_.startCase(_.capitalize(langName))]
```

Comment:
@software-project if you switch to capitalize then you shouldn't need the _.startCase